src/main/java/org/folio/bulkops/controller/BulkOperationController.java

```diff
@@ -161,4 +161,11 @@ public ResponseEntity<Void> deleteFileByNameAndOperationId(UUID operationId, Str
     logFilesService.deleteFileByOperationIdAndName(operationId, fileName);
     return new ResponseEntity<>(HttpStatus.NO_CONTENT);
   }
+
+  @Override
+  public ResponseEntity<Void> cancelOperationById(UUID operationId) {
+    log.info("Cancelling bulk operation id={}", operationId);
```

Comment:
use the log level debug, pls.
According to guide
https://wiki.folio.org/display/DD/Logging#Logging-1.2Businesscodelogging